commons-vfs2/src/main/java/org/apache/commons/vfs2/cache/SoftRefFilesCache.java

```diff
@@ -43,85 +38,55 @@
  */
 public class SoftRefFilesCache extends AbstractFilesCache {
 
-    private static final int TIMEOUT = 1000;
-
     private static final Log log = LogFactory.getLog(SoftRefFilesCache.class);
 
-    private final ConcurrentMap<FileSystem, Map<FileName, Reference<FileObject>>> fileSystemCache = new ConcurrentHashMap<>();
+    private final Map<FileSystem, Map<FileName, Reference<FileObject>>> fileSystemCache = new HashMap<>();
     private final Map<Reference<FileObject>, FileSystemAndNameKey> refReverseMap = new HashMap<>(100);
     private final ReferenceQueue<FileObject> refQueue = new ReferenceQueue<>();
 
-    private volatile SoftRefReleaseThread softRefReleaseThread = null; // @GuardedBy("lock")
-
-    private final Lock lock = new ReentrantLock();
+    private SoftRefReleaseThread softRefReleaseThread = null;
```

Comment:
Please don't initialize to default values.

Reply:
Agree, but not my code / choice. I only removed the `volatile` and the obsolete code comment. I preserved the old code as-is, and I think refactoring to remove unnecessary default values should not be mixed in this commit.